src/server/scripts/Custom/Solocraft.cpp

```diff
@@ -386,8 +410,24 @@ class solocraft_player_instance_handler : public PlayerScript {
         }
         return numInGroup;
     }
+    // Get the Player's class balance debuff
+    uint32 GetClassBalance(Player* player)
```

Comment:
This method could be optimized by storing what .find() returns and using that instead of doing a lookup in classes 3 times. There are only a few classes so it's fine....

Reply:
@acidmanifesto you could deal with this comment in another PR if you want to train your C++

Reply:
i will definately look into it.